activestate.yaml

```diff
@@ -138,17 +138,17 @@ scripts:
     value: |
       set -e
       echo "Building State Tool"
-      $scripts.build.path()
+      $scripts.build.path.posix()
```

Comment:
Please use the underscore variant we discussed, ie. `_posix()`. The underscore is important as it communicates "danger".